indico/modules/rb_new/schemas.py

```diff
@@ -19,10 +19,13 @@
 from marshmallow.fields import Nested, String
 
 from indico.core.marshmallow import mm
+from indico.modules.rb import Blocking
```

Comment:
i'd import that from `indico.modules.rb.models.blockings` like we do for other models - the fact that it's also available in `__init__.py` is just because we use it in there